tests/libs/generate_api_v2_test.py

```diff
@@ -125,11 +134,38 @@ def test_build_summary_for_fips(
                 ],
                 anomalies=[],
             ),
-            caseDensity=FieldAnnotations(sources=[field_source_usafacts], anomalies=[]),
+            caseDensity=FieldAnnotations(
+                sources=[field_source_usafacts],
+                anomalies=[
+                    {
+                        "date": datetime.date(2021, 2, 25),
+                        "original_observation": 102666.0,
+                        "type": "cumulative_tail_truncated",
+                    }
+                ],
+            ),
             icuCapacityRatio=FieldAnnotations(sources=[field_source_hhshospital], anomalies=[]),
             icuHeadroomRatio=FieldAnnotations(sources=[field_source_hhshospital], anomalies=[]),
-            infectionRate=FieldAnnotations(sources=[field_source_usafacts], anomalies=[]),
-            infectionRateCI90=FieldAnnotations(sources=[field_source_usafacts], anomalies=[]),
+            infectionRate=FieldAnnotations(
+                sources=[field_source_usafacts],
+                anomalies=[
+                    {
+                        "date": datetime.date(2021, 2, 25),
+                        "original_observation": 102666.0,
+                        "type": "cumulative_tail_truncated",
+                    }
+                ],
+            ),
+            infectionRateCI90=FieldAnnotations(
```

Comment:
hmm interesting case - I'm not sure we want to pass the anomalies down to these fields.  Probably only want the source